gatsby-node.js

```diff
@@ -86,8 +86,8 @@ exports.onCreateNode = ({ node, actions, getNode, createNodeId, createContentDig
 
 exports.createPages = async ({ actions, graphql, reporter }) => {
   const { createPage } = actions;
-  const projectTemplate = require.resolve(`./src/components/project.tsx`);
-  const projectsTemplate = require.resolve(`./src/components/projects.tsx`);
+  const projectTemplate = require.resolve(`./src/components/projectTemplate.tsx`);
```

Comment:
I just renamed the templates for project so that the new `Project` component didn't have a name conflict. We should probably create a `templates` folder at some stage so that they're not just in the components folder.

Reply:
I made a templates folder for them now, not too big of a deal to do